presto-main/src/main/java/io/prestosql/server/ServerMainModule.java

```diff
@@ -324,6 +336,22 @@ protected void setup(Binder binder)
         // execution
         binder.bind(LocationFactory.class).to(HttpLocationFactory.class).in(Scopes.SINGLETON);
 
+        // dynamic filter summary
```

Comment:
Question: would it be easier to provide the required bindings in a separate module (e.g. `DynamicFilteringModule`), which can be then added to `PrestoServer` and `TestingPrestoServer`?
This way, we can (hopefully) keep `ServerMainModule` the same...
WDYT?

Reply:
I've now made the changes to add the required bindings through `DynamicFilteringClientModule` and `TestingDynamicFilteringClientModule` and avoided the changes to `ServerMainModule`